docs/data/joy/components/alert/alert.md

```diff
@@ -27,28 +36,57 @@ export default function MyApp() {
 }
 ```
```

Comment:
@samuelsycamore I think this can be simplified a bit more. How about having just the import example for the **Usage** because the **Basics** has covered the render part.

```js
import Alert from '@mui/joy/Alert';
```

Reply:
I think it's nice to have the full context in a "Quickstart" style here. Instead, as @oliviertassinari has suggested, I'm going to expand the Basics section to provide a better demo that the user could copy and paste, so these two sections will be a bit less redundant.